test/command_line/suite/grndb/test_options.rb

```diff
@@ -0,0 +1,71 @@
+# Copyright(C) 2019 Kouhei Sutou <kou@clear-code.com>
+# Copyright(C) 2019 Kentaro Hayashi <hayashi@clear-code.com>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 2.1 as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+class TestGrnDBOptions < GroongaTestCase
+  sub_test_case("--log-flags") do
+    test("default") do
+      groonga("status")
+      grndb("check")
+      assert_equal("1970-01-01 00:00:00.000000|n| " +
+                   "grn_init: <VERSION>",
+                   normalize_init_line(File.readlines(@log_path).first))
+    end
+
+    test("add: one") do
+      flags = "+pid"
+      groonga("status",
+              command_line: ["--log-flags", flags])
```

Comment:
Why is this option is needed for `groonga`?

Reply:
Ah, It is needless. And I've noticed that test case is wrong. I've fixed.